### ISSUE TITLE
Running the example notebook

### ISSUE THREAD

**Issue by p16i** [understandable-machine-intelligence-lab/NoiseGrad#1]
Hi,

I've slightly modified the notebook such that I could run it without the need to accessing Google Drive.

However, it seems that the result I get is quite different from yours:

This is my result
![image](https://user-images.githubusercontent.com/1214890/126064556-eae1f5c0-db53-4e8b-91c3-1437cb91b9dc.png)

This is your result
![](https://github.com/understandable-machine-intelligence-lab/NoiseGrad/raw/master/samples/resulting_explanation.png)

Do you have any idea what could go wrong? 

One thing that could contribute to the problem is that the input image is quite different from your results. In fact, **the original image in your repo has quite high contrast**.

![](https://github.com/understandable-machine-intelligence-lab/NoiseGrad/blob/master/samples/lama_image.png?raw=true)

**Comment by lapalap** [comment]
Hey!

Thank you for your comment! 
Indeed there was a bug where the image that we had in our repo was saved already normalized (this is why it looks like it has high contrast). For example notebook, we now uploaded an un-normalized image and perform normalization explicitly in the script. So, I believe, the difference between your result and ours is because you applied normalization twice.

We have also deleted all connections to collab, so it is now possible to run the sample script on any machine!
